addons/pos_customer_display/static/src/app/root.xml

```diff
@@ -2,33 +2,40 @@
                 </div>
-                <div t-if="!order.finalized and order.lines?.length" class="m-2 p-3 rounded-3" t-attf-style="background-color: {{bgColor}};">
-                    <h3 class="text-white">Your Order</h3>
+                <div class="d-flex align-items-center ps-3 pe-2 py-1 rounded-3 text-bg-dark">Powered by <OdooLogo style="'width: 4rem;'" monochrome="true"/></div>
+            </div>
+            <div class="o_customer_display_main d-flex flex-column flex-grow-1 overflow-auto">
+                <div t-if="!order.finalized and order.lines?.length" class="d-flex px-4 py-1 text-bg-dark">
+                    <h3 class="text-reset pt-2">Your Order</h3>
                 </div>
-                <OrderWidget t-if="!order.finalized" lines="order.lines" t-slot-scope="scope" style="'background-color: #f6f6f6;'">
-                    <Orderline line="scope.line" class="'bg-white mx-3 my-1 rounded-3' + (scope.line.isSelected ? ' selected' : '')"/>
+                <OrderWidget t-if="!order.finalized" lines="order.lines" t-slot-scope="scope" class="'bg-300'" style="'scroll-snap-type: y mandatory;'">
+                    <Orderline line="scope.line" imgSize="'4rem'" infoListClasses="''" class="'o_customer_display_orderline d-flex align-items-center mx-2 my-1 ps-1 pe-3 rounded-3 bg-white fs-3' + (scope.line.isSelected ? ' selected' : '')"/>
                 </OrderWidget>
-                <CenteredIcon t-else="" icon="'fa-smile-o fa-4x'" text="'Thank you'"/>
-                <div class="text-white p-2 px-5 d-flex flex-column justify-content-center" t-attf-style="background-color: {{bgColor}};">
-                    <div class="row">
-                        <h4 class="col text-start text-white">Total</h4>
-                        <div class="col text-end" t-esc="order.amount"/>
-                    </div>
-                    <div t-foreach="order.paymentLines or []" t-as="line" t-key="line_index" class="row">
-                        <div class="col text-start" t-esc="line.name" />
-                        <div class="col text-end" t-esc="line.amount"/>
-                    </div>
-                    <div class="row" t-if="order.change">
-                        <div class="col text-start">Change</div>
-                        <div class="col text-end" t-esc="order.change"/>
-                    </div>
-                    <div class="d-flex align-items-center ms-auto">Powered by<OdooLogo style="'width: 4rem;'" monochrome="true" /></div>
+                <CenteredIcon t-else="" icon="'fa-smile-o fa-5x'" text="'Thank you'"/>
+                <div class="w-100 text-bg-dark">
+                    <table t-if="order.amount" t-attf-class="table table-dark table-borderless table-sm w-75 w-sm-50 w-xl-25 m-3 ms-md-auto ms-auto text-end">
+                        <thead>
+                            <tr class="fs-2">
+                                <th scope="col">Total</th>
```

Comment:
The labels here are moving when we add content. Can't we fix the size of the columns (at least with a min-width) ?
https://tinyurl.com/4c6497nm